common/templates/general.go

```diff
@@ -716,14 +718,11 @@ func shuffle(seq interface{}) (interface{}, error) {
 		return nil, errors.New("can't iterate over a nil value")
 	}
 
-	switch seqv.Kind() {
-	case reflect.Array, reflect.Slice, reflect.String:
-		// okay
-	default:
+	if seqv.Kind() != reflect.Slice {
```

Comment:
This was changed as shuffle never worked with strings or arrays in the first place.
MakeSlice panics if the passed Type does not have kind Slice.